src/overlays/actors/ovl_En_Toto/z_en_toto.c

```diff
@@ -452,30 +452,31 @@ s32 func_80BA4204(EnToto* this, PlayState* play) {
 
 s32 func_80BA42BC(EnToto* this, PlayState* play) {
     Player* player = GET_PLAYER(play);
-    u32 phi_s0 = 0;
-    Vec3s* end = &D_80BA510C[3];
+    u32 numPoints = 0;
+    Vec3s* endPosListPtr = &sPlayerOverrideInputPosList[ARRAY_COUNT(sPlayerOverrideInputPosList)];
```

Comment:
Nice

Reply:
Would this technically be an OoB read? I think it's fine right? Just jank?

Reply:
Yeah should just be jank. It never actually reads from the end of the array, just gets the pointer.